background.js

```diff
@@ -4,7 +4,7 @@
 browser.browserAction.onClicked.addListener(runLightbeam);
 
 async function runLightbeam() {
-
+  captureData();
```

Comment:
This prevents the add-on from capturing any data unless the browserAction icon is clicked. Is that what we want?